Adding query for all repos for an org or user

jimbobbennett opened the pull request:
## Description

Adds polling for the repos belonging to an org, and allows them to be used instead of a list of specific repos.

## Fixes

#6 

## Type of PR
- [ ] Bug fix
- [X] New feature
- [ ] Breaking change

## Checklist
- [X] Code follows this projects style guide
- [X] I reviewed my code
- [ ] Tested in deployment environment
- [X] Tested in an alternate environment (please update here: tested locally)
- [X] Required documentation has been updated
- [ ] Other (explain: )

## Additional information

jimbobbennett committed:
Adding query for all repos for an org or user

adiazulay reviewed (commented):
This looks awesome! I really like that is will auto update if you make new repos.

I added a suggestion for improving the encapsulation by passing the base url in the __init__ methods. Let me know what you think.

jimbobbennett committed:
PR fixes

jimbobbennett commented on README.md:
````diff
@@ -142,15 +142,32 @@ You can use the Data Explorer in the Azure portal to create a database and conta
+            repos = org.repos()
+        ```
+
+        Replace `<USER-OR-ORG-OWNER>` with the GitHub user name or organization name.
````
Fixed manually with other tweaks

jimbobbennett commented on github_tracker/traffic_tracker.py:
```diff
@@ -70,7 +106,6 @@ class Repo:
-    _API_URL_BASE = "https://api.github.com"
     _V4 = "/graphql"
 
     def __init__(self, owner, name, url, api_token):
```
I'm not sure this really adds much - the `API_URL_BASE` won't ever change unless there is a new API which would need bigger changes, so why pass it in?

As a suggestion, I've renamed it to _API_URL_BASE to indicate that it's private.

jimbobbennett commented on github_tracker/__init__.py:
```diff
@@ -14,6 +14,10 @@
 
 
 def main(mytimer: func.TimerRequest):
+    # Uncomment this to load the repos from GitHub each time
```
Moved

jimbobbennett commented on github_tracker/traffic_tracker.py:
```diff
@@ -46,6 +47,41 @@ def upload(self, data):
         '''
         self._container_client.upsert_item(data)
 
+class Org:
```
Renamed to UserOrOrg

jimbobbennett commented:
Fixed a few suggestions. Not sure the encapsulation adds anything except potential for confusion around what should be passed in. This value will never change, so no reason to have it set manually instead of using a constant. I've added an _ to the start to indicate it's private.

jimbobbennett committed:
Fixing naming

jimbobbennett committed:
Update README.md

jimbobbennett committed:
Fixing naming

jimbobbennett committed:
Update __init__.py

adiazulay commented:
Looks good, tested this on my device and everything works.
